ui/src/main/js/common/input/EndpointSelectField.js

```diff
@@ -36,16 +36,6 @@ const EndpointSelectField = ({
     const [options, setOptions] = useState([]);
     const [requestErrorValue, setRequestErrorValue] = useState(null);
 
-    const emptyFieldValue = async () => {
-        const eventObject = {
-            target: {
-                name: fieldKey,
-                value: []
-            }
-        };
-        onChange(eventObject);
-    };
```

Comment:
This seemed to do nothing but break the field, if someone knows what this prevents let me know